includes/classes/order.php

```diff
@@ -387,6 +387,7 @@ function query($order_id)
         if (IS_ADMIN_FLAG === true) {
             $this->notify('ORDER_QUERY_ADMIN_COMPLETE', ['orders_id' => $this->orderId]);
         }
+        return true;
```

Comment:
why is this necessary?

Reply:
Good to see you are still on quality control.

This method returns false in places, so it seemed correct to return true at the end of the parsing,prior to hinting the method.
However I find no instances of this returned value being checked anywhere....